src/cartridges/int_adyen_controllers/cartridge/controllers/Adyen.js

```diff
@@ -532,10 +540,11 @@ function authorize3ds2() {
             'utm_nooverride',
             '1',
           ),
-          resultCode: result.resultCode,
-          token3ds2: result.authentication['threeds2.challengeToken'],
+          action: JSON.stringify(result.action),
```

Comment:
This was giving an error on SFRA for some cases "URL too long". Is this properly working for SG?

Reply:
Yes for SG it doesn't cause any problems